src/data/int/cast.lean

```diff
@@ -177,23 +181,53 @@ by rw [← cast_zero, cast_lt]
 @[simp] theorem cast_lt_zero [ordered_ring α] [nontrivial α] {n : ℤ} : (n : α) < 0 ↔ n < 0 :=
 by rw [← cast_zero, cast_lt]
 
-@[simp, norm_cast] theorem cast_min [linear_ordered_ring α] {a b : ℤ} :
+section linear_ordered_ring
+
+variables [linear_ordered_ring α]
+
+@[simp, norm_cast] theorem cast_min {a b : ℤ} :
```

Comment:
Hmm, I think these are OK being implicit.

Also I don't want to change too much as I go --- I'm only touching these lemmas because I introduced a `variables [linear_ordered_ring α]` to tidy up their statements.

Reply:
No problem, I wasn't sure either.